server/model/events/events.go

```diff
@@ -396,3 +396,18 @@ func TestSpecsAssertionWarning(testID id.ID, runID int, err error, spanID string
 		Outputs:             []model.OutputInfo{},
 	}
 }
+
+func TraceOtlpServerReceivedSpans(testID id.ID, runID int, spanCount int) model.TestRunEvent {
+	return model.TestRunEvent{
+		TestID:              testID,
+		RunID:               runID,
+		Stage:               model.StageTrace,
+		Type:                "OTLP_SERVER_RECEIVED_SPANS",
+		Title:               "OTLP server received spans",
+		Description:         fmt.Sprintf("The Tracetest OTLP server received %d spans", spanCount),
```

Comment:
Does it add any value to include if it was HTTP or GRPC?

Reply:
Good idea, I just added it to the event